src/app/icd/ICDManager.cpp

```diff
@@ -50,9 +50,13 @@ void ICDManager::Init(PersistentStorageDelegate * storage, FabricTable * fabricT
 
     bool supportLIT = SupportsFeature(Feature::kLongIdleTimeSupport);
     VerifyOrDieWithMsg((supportLIT == false) || SupportsFeature(Feature::kCheckInProtocolSupport), AppServer,
-                       "The CheckIn protocol feature is required for LIT support");
+                       "The CheckIn protocol feature is required for LIT support.");
```

Comment:
With the increasing number of checks in LIT, maybe it would be better to remove the `(supportLIT == false) ||` from all of them and just do the series of checks  if supportLIT == true`

Reply:
done